### DISCUSSION TITLE
Is there any way to unprune a zombie channel?

### DISCUSSION THREAD

**Discussion by hazrulnizam** [lightningnetwork/lnd#7088] [Q&A] [answered]
I was having trouble sending some sats to a node with a NO_ROUTE error. However, I know that a route exists, and there is enough liquidity. The route goes like this:

```
Node A <--(chanID1)--> Node B <--(chanID2)--> Node C
```
I am Node A sending to Node C.
Checking from Node A, I get the following:
```shell
$ lncli getchaninfo chanID2
[lncli] rpc error: code = Unknown desc = edge marked as zombie
```
It seems like this edge was pruned and marked as a zombie by my node. I know for a fact that this channel is active as I have another node that gives out the correct info to the above command.

Using online search, I was able to find a way to overcome this: using @guggero's [chantools dropchannelgraph](https://github.com/guggero/chantools/blob/master/doc/chantools_dropchannelgraph.md).

This however, required my node to rebuild the whole graph from scratch, which took a while. Ultimately it fixed the issue though, and I was able to send the payment.

My question, though, is: Is there a simpler way to do this, other than to rely on an unsupported method that destroys the whole graph? Can I 'unmark' an edge from being a zombie?

**Comment by guggero** [comment, marked as answer]
If the channel is indeed active, then its channel update message should have resurrected it from being a zombie. Maybe those messages (e.g. channel is active) didn't reach your node for some reason? Or maybe the resurrection doesn't work correctly?

**Reply by hazrulnizam** [reply]
I see. This is possible, as the node I was talking about was running without any active channels for a while, thus perhaps disconnected from active gossip. Perhaps if I have waited longer after reconnecting it to the network, it might have gotten new updates and resurrected the zombie as you said. I did wait for around 2 hours initially, but grew impatient and looked for workarounds.

Thanks for the quick answer!
